package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kakunin",
-  "version": "2.4.1",
+  "version": "2.5.0",
```

Comment:
Don't change the version manually, it will be changed via npm command in the separated PR.